backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Net.java

```diff
@@ -73,14 +73,21 @@ public boolean openURI (String uri) {
 			} catch (Throwable t) {
 				return false;
 			}
-		} else {
+		} else if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
 			try {
 				Desktop.getDesktop().browse(new URI(uri));
 				return true;
 			} catch (Throwable t) {
 				return false;
 			}
+		} else if (SharedLibraryLoader.isLinux) {
+			try {
+				(new ProcessBuilder("xdg-open", (new URI(uri).toString()))).start();
```

Comment:
Maybe `new URI(uri).toString()` can just be `url`?

Reply:
This appears messy, but it has a purpose. Running the `uri` through a `URI` builder sanitizes the input to prevent arbitrary code execution on a client system by other players in online environments.

See https://github.com/libgdx/libgdx/pull/6543#issuecomment-848766738 for details

Reply:
That makes sense. Good to keep it!